E0253 needs to be updated to new format

From: src/test/compile-fail/E0253.rs

E0253 needs a span_label, updating it from:

```
error[E0253]: `do_something` is not directly importable
  --> src/test/compile-fail/E0253.rs:17:5
   |
17 | use foo::MyTrait::do_something; //~ ERROR E0253
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

To:

```
error[E0253]: `do_something` is not directly importable
  --> src/test/compile-fail/E0253.rs:17:5
   |
17 | use foo::MyTrait::do_something; //~ ERROR E0253
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot be imported directly
```

--- a/src/librustc_resolve/resolve_imports.rs
+++ b/src/librustc_resolve/resolve_imports.rs
@@ -505,7 +505,9 @@ impl<'a, 'b:'a> ImportResolver<'a, 'b> {
                 }
                 Success(binding) if !binding.is_importable() => {
                     let msg = format!("`{}` is not directly importable", target);
-                    span_err!(self.session, directive.span, E0253, "{}", &msg);
+                    struct_span_err!(self.session, directive.span, E0253, "{}", &msg)
+                        .span_label(directive.span, &format!("cannot be imported directly"))
+                        .emit();
                     // Do not import this illegal binding. Import a dummy binding and pretend
                     // everything is fine
                     self.import_dummy_binding(module, directive);

--- a/src/test/compile-fail/E0253.rs
+++ b/src/test/compile-fail/E0253.rs
@@ -16,6 +16,6 @@ mod foo {
 
 use foo::MyTrait::do_something;
     //~^ ERROR E0253
-    //~|NOTE not directly importable
+    //~|NOTE cannot be imported directly
 
 fn main() {}

--- a/src/test/compile-fail/E0253.rs
+++ b/src/test/compile-fail/E0253.rs
@@ -14,6 +14,8 @@ mod foo {
     }
 }
 
-use foo::MyTrait::do_something; //~ ERROR E0253
+use foo::MyTrait::do_something;
+    //~^ ERROR E0253
+    //~|NOTE not directly importable
 
 fn main() {}

